### ISSUE TITLE
Update of the DB is not reflected

### ISSUE THREAD

**Issue by sandeepjoshiaw** [project-imas/encrypted-core-data#221]
In my project, I have two processes which are interacting with the Core Data. One of the process is inserting data and also updating it. Now I fetch the DB using another process, I get the Data.

But when I update the DB using one process while another is running and fetch the data from another process, I don't see the changes. I get the old data. However, If I kill the process and run again, I can get the updated data.

**Comment by dorada** [comment]
Multithreaded core data is tricky, does this work with the standard stack?

**Comment by sandeepjoshiaw** [comment]
Yes the DB operations I mentioned works fine without the encryption and FYI I am using this in mac.

**Comment by dorada** [comment]
Can you make/modify a unit test to reproduce this?
